src/schemas/json/bacalhau-job.json

```
@@ -0,0 +1,506 @@
+{
+  "$defs": {
+    "Deal": {
+      "properties": {
+        "Concurrency": {
+          "type": "integer"
+        },
```

```suggestion
        "Concurrency": {
          "description": "description\ndoc_link",
          "type": "integer"
        },
```

Optional fix: description and a link to docs after a trailing new line can be added.